workflow.json

```diff
@@ -4,32 +4,37 @@
     "parameters": {
         "logic_app_name": {
             "defaultValue": "",
-            "type": "string"
+            "type": "String"
```

Comment:
not valid according to the template spec, but the ARM API is returning it in uppercase to TF so it causes a lot of diffs without it